score-client/src/main/resources/application.yml

```diff
@@ -69,49 +69,17 @@ portal:
     
 logging:
   file: target/logs/client.log
+  level:
+    bio.overture.score.client: DEBUG
```

Comment:
I don't think we want to leave this in our release.

Reply:
removed.